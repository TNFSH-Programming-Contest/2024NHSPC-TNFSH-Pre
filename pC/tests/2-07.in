99 554
38 92
14 32
34 21
65 10
50 96
9 5
80 58
21 94
24 56
45 30
73 52
1 97
3 56
18 93
54 73
86 79
53 18
56 19
9 26
56 54
28 85
51 85
48 69
75 88
53 34
97 37
76 82
57 81
19 75
6 34
64 58
1 19
78 60
17 38
81 90
81 94
60 91
38 12
18 12
54 44
39 53
21 9
88 57
53 35
36 29
63 90
97 52
68 27
86 56
57 1
86 71
86 43
38 5
84 72
98 69
87 35
20 10
29 5
16 58
37 11
70 67
68 67
65 6
99 77
11 43
21 81
18 44
14 2
25 84
51 13
75 70
90 5
8 17
6 26
80 56
40 91
62 29
30 73
29 61
57 2
17 60
5 55
43 30
70 49
75 63
58 17
71 53
45 46
8 57
45 84
57 45
61 91
33 2
68 70
33 54
67 32
16 59
24 93
16 49
72 93
93 43
27 21
44 31
12 76
20 40
38 40
41 50
53 25
10 54
49 11
76 83
54 35
53 72
29 60
90 14
95 36
89 25
68 17
48 98
95 41
36 17
33 83
72 97
17 21
20 37
56 53
76 97
31 71
80 8
6 59
75 34
41 16
53 58
8 68
44 29
82 95
52 49
77 78
59 24
5 8
69 25
47 82
5 96
37 25
62 83
78 87
89 67
83 38
46 28
26 87
12 21
69 33
32 95
83 53
69 13
16 26
94 85
92 55
10 6
35 92
41 93
85 52
82 50
16 86
75 85
62 66
50 8
16 45
56 38
19 37
65 81
4 44
92 4
73 24
55 87
58 50
12 73
90 26
75 36
94 24
87 89
19 62
95 39
7 95
95 62
91 15
73 59
7 41
87 4
46 98
42 9
10 71
81 36
94 77
44 81
95 35
8 19
4 47
32 71
75 53
87 69
2 60
67 76
62 49
45 79
67 97
56 23
90 98
4 1
97 68
30 93
18 14
22 21
35 47
43 69
69 47
6 72
38 74
41 71
28 55
93 87
95 59
91 20
42 27
1 8
86 53
68 25
43 52
29 57
67 1
98 84
5 25
77 11
66 49
85 87
45 66
70 60
93 78
36 69
6 50
33 77
79 64
5 17
34 93
24 85
1 2
8 51
47 97
16 4
61 56
52 33
76 54
80 76
98 43
67 98
57 28
16 87
24 70
18 70
65 33
6 44
9 50
61 33
4 7
95 94
34 15
37 82
14 30
99 64
31 3
64 20
42 43
99 60
31 21
3 32
71 12
7 3
36 32
12 2
14 59
74 21
37 34
28 75
59 49
27 91
75 46
67 69
72 7
88 42
19 45
41 86
39 17
63 69
1 96
77 74
4 22
26 54
22 25
26 8
17 67
79 46
82 23
94 46
21 19
86 62
48 28
11 22
19 28
63 45
9 76
50 84
71 72
33 16
75 42
59 91
56 70
71 90
50 95
76 42
84 12
7 15
17 59
15 17
76 31
49 92
8 31
63 37
52 81
2 83
31 59
43 2
32 44
90 23
91 33
39 59
26 50
92 10
38 53
17 52
72 45
67 43
2 19
31 93
6 53
91 34
50 11
64 48
87 3
71 35
94 64
6 5
2 36
85 82
46 13
62 31
17 48
84 74
10 8
72 75
2 39
94 50
29 64
56 97
50 54
27 14
45 75
13 60
74 42
78 36
9 15
71 2
19 25
24 9
45 82
44 5
95 40
74 78
10 38
98 36
80 12
18 32
34 95
72 29
3 30
44 14
35 97
14 87
11 1
3 48
17 28
65 59
70 53
41 33
13 70
19 49
75 37
69 10
47 88
76 84
4 35
21 33
91 50
70 96
13 65
19 36
76 34
19 92
44 95
96 23
31 38
62 88
69 17
5 22
94 26
99 94
39 46
37 3
68 51
88 94
22 92
78 15
74 71
94 7
32 30
57 21
56 63
13 43
80 5
35 49
25 11
74 6
44 98
84 49
11 29
5 71
57 69
46 83
93 95
78 5
17 97
31 35
18 55
65 94
57 76
60 14
53 1
74 82
16 44
5 86
74 41
4 37
63 9
54 45
77 84
78 95
32 4
1 44
51 32
88 28
35 60
89 88
90 35
39 16
97 77
21 23
35 12
19 93
71 28
3 1
90 36
9 91
58 35
92 21
72 20
89 42
97 59
38 27
97 44
90 29
6 18
7 38
59 26
4 77
37 73
20 63
72 87
12 78
60 40
18 62
66 9
60 48
78 67
99 34
85 1
36 56
99 56
9 2
60 34
23 3
16 29
82 91
84 46
51 37
17 35
69 64
80 62
32 5
20 47
41 6
79 51
11 14
76 55
8 40
86 73
69 26
7 64
37 55
13 3
47 14
79 65
28 40
23 99
85 34
69 42
12 5
55 82
30 7
10 28
19 86
27 10
5 2
49 90
41 67
33 25
63 3
56 47
90 6
10 32
14 9
34 69
11 52
65 56
41 92
67 79
37 6
63 36
80 75
70 51
5 33
48 8
65 40
22 31
51 64
46 7
