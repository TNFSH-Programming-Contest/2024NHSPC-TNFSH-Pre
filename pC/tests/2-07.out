54
7 11 12 13 15 18 20 22 23 24 25 26 27 29 30 31 39 40 41 42 43 47 48 51 52 54 55 57 58 61 63 64 65 66 68 72 73 74 77 78 79 80 81 82 83 84 85 86 88 89 92 96 98 99 